encoding/delta/byte_array.go

```diff
@@ -1,16 +1,16 @@
 package delta
 
 import (
-	"bytes"
 	"fmt"
+	"sort"
 
 	"github.com/segmentio/parquet-go/encoding"
 	"github.com/segmentio/parquet-go/encoding/plain"
 	"github.com/segmentio/parquet-go/format"
 )
 
 const (
-	maxLinearSearchPrefixLength = 128 // arbitrary
+	maxLinearSearchPrefixLength = 64 // arbitrary
```

Comment:
Since the new algorithm scales a lot better I lowered the threshold based on observations from the benchmarks we have in the codebase.